Tasks/Maven/maventask.ts

```diff
@@ -212,11 +213,17 @@ function createMavenSQRunner(sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?
 function applyPmdGoals(mvnRun: trm.ToolRunner):void {
     if (tl.getInput('pmdAnalysisEnabled', true) != 'true') {
         console.log('PMD analysis is not enabled');
+        return;
     }
     console.log('PMD analysis is enabled');
     pmd.applyPmdArgs(mvnb);
 }
 
+// Take a result object and create one line of the build summary from it.
+function buildSummaryLineFromResult(analysisResult:ar.AnalysisResult):string {
+    return `${analysisResult.toolName} found ${analysisResult.totalViolations} violations in ${analysisResult.filesWithViolations} files.`;
```

Comment:
Although the Maven task does not currently use localization, it's probably because Maven will anyway output messages in English (I suppose, I haven't checked!). However, if we are going to create a summary, it would be best if it's localized. See https://github.com/Microsoft/vsts-task-lib/blob/master/node/docs/vsts-task-lib.md on how to localize strings. Also take a look at the npm task that is localized. I suggest you localize only the strings related to the summary for now, and then you can chat with David and Yang to see if we should localize everything else.